lib/reply.js

```diff
@@ -66,6 +67,7 @@ function Reply (res, request, log) {
   this[kReplyTrailers] = null
   this[kReplyHasStatusCode] = false
   this[kReplyStartTime] = undefined
+  this[kReplyPayload] = undefined
```

Comment:
Only concern is when the payload is `stream`.
How can the subscriber safely consume the `stream` for logging?
The same problem exist when I implement the `Trailer` feature and it is not resolved until now.

Reply:
I can see so many ways this can end badly long term. Can you avoid storing the payload inside reply?

Reply:
The latest commit no longer assigns the payload value to the `Reply` instance: 325ba542cd19759cf18eb69b6b11c116e462f70e